lib/src/components/NxLoadError/NxLoadError.scss

```diff
@@ -0,0 +1,21 @@
+.nx-load-error__message {
+  // TODO no magic numbers
+  max-width: calc(100% - 32px);
```

Comment:
I noticed that in cases where the button wraps down (very clever solution BTW), but the text is long enough to read the right edge of the box it doesn't look very good.
<img width="576" alt="Screen Shot 2020-06-11 at 2 32 19 PM" src="https://user-images.githubusercontent.com/1991443/84427201-84ff9880-abf2-11ea-9cf2-ff727cf949b8.png">
However if you add 8px of margin-bottom to `.nx-load-error__message` it adds some breathing space but doesn't effect non-wrapping instances.
<img width="400" alt="Screen Shot 2020-06-11 at 2 46 07 PM" src="https://user-images.githubusercontent.com/1991443/84427291-afe9ec80-abf2-11ea-9f05-63b92bf71d11.png">

Reply:
> However if you add 8px of margin-bottom to .nx-load-error__message it adds some breathing space but doesn't effect non-wrapping instances.

631fb79